packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1630,18 +1630,18 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
             switch (message.type) {
                 case ContainerMessageType.Attach:
-                    this.dataStores.processAttachMessage(message, local || localAck);
+                    this.dataStores.processAttachMessage(message, localAck);
```

Comment:
I'm not sure what's the difference between local & localAck. Any chance you can add comments in the code to make it clearer for readers?

Reply:
ping

Reply:
I added a comment. I have been working on a bigger PR like you suggested and localAck will go away since we won't see already-ACKed ops

Reply:
Ah, Ok. Ping me when you want me to take a look, I just want to make sure I do not block you :)